src/style.css

```
@@ -1 +1,68 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.menu-link__logo {
+  padding: 10px 0;
+}
+
+/* revome margin-bottom 3px that appears when img is inside a*/
+.menu-link__logo > img {
+  display: block;
+}
+
+.menu {
+  display: flex;
+  list-style: none;
+}
+
+.menu-li:not(:first-child) {
+  margin-left: 20px;
+}
+
+.menu-link {
+  color: black;
+  font-size: 1rem;
+  text-decoration: none;
+  font-weight: 500;
+  text-transform: uppercase;
+  padding: 23px 0;
+  line-height: 14px;
+  display: inline-block;
```

Why inline-block? You don't need it here
```suggestion
  display: block;
```